design/one-pager-managed-resource-api-design.md

```diff
@@ -51,12 +51,27 @@ deviate so that we can reassess and improve it.
 
 ## Guiding Principles
 
-Our users are the stakeholders on how we make decisions. There are two types of users for managed resources:
-* Input Users: Users who configure/create/manage the resource. `Crossplane Claim` Controller, `Cluster Admin` are a few
-  examples of this type. We want our `Spec` to be the source of truth for them to configure everything about the resource.
-* Output Users: Users who refer to/query/use/extract information from the managed resource. We want them to be able to
-  perform all read-only operations on the given resource as if they have the cloud provider's API available. This can
-  include configurations and metadata information about the resource.
+Our users are the stakeholders on how we make decisions. There are two
```

Comment:
Nit: "the stakeholders on how we make decisions" makes sense, but reads a little awkwardly to me. What do you think about "Our users are our stakeholders. They influence how we make decisions."?

Reply:
SGTM! I actually didn't modify this part but it got rewrapped so shows it as a change so want to check with @muvaf before I update 👍

Reply:
I'm OK with new phrase but I'd not partially rewrap only this section and the added section. Maybe another PR or commit to rewrap all if that's really necessary?

Reply:
@muvaf I am not sure if it is necessary or not, but I was unable to determine what the current wrapping was. It did not seem like it was a uniform character limit. Do you know?

Reply:
I went ahead and added a commit to rewrap at 80 chars